games/maze/src/maze.rs

```diff
@@ -11,6 +11,14 @@ pub struct Maze {
     map: Vec<Tile>,
 }
 
+#[derive(Debug, Serialize)]
+pub struct GiveDirections {
```

Comment:
How can a struct Give directions? The name does not make a lot of sense. Instead you should call it what it is such as `NeighbouringTileTypes`.